dotCMS/src/main/java/com/dotcms/business/SystemTableFactoryImpl.java

```
@@ -114,4 +117,13 @@ public void delete(final String key) throws DotDataException {
     public void clearCache() {
         CacheableEagerFactory.super.clearCache();
     }
+
+    @Override
+    public void initIfNeeded() {
+
+        Try.run(()->new DotConnect().executeStatement("CREATE TABLE if not exists system_table ("
+                + "key varchar(511) primary key,"
+                + "value text not null"
+                + ")"));
```

I would throw here if this fails rather than eat the exception.